### DISCUSSION TITLE
Map RFC1918 IPs to private ASN

### DISCUSSION THREAD

**Discussion by eahlys** [akvorado/akvorado#490] [Q&A]
Hi,

I have a few RFC1918 IPv4 subnets which are currently seen by Akvorado as "AS0". I would like to remap those subnets to other ASNs (like 192.168.0.0/16 = AS64501 ; 10.0.0.0/0 = AS64502). I think there is a way to do this but I can't find it. Do you have any clue ? :) 

- Pierre from Shadow

**Comment by vincentbernat** [comment]
Currently, this is only possible with when ASN are embedded into flows or when retrieved through BMP.

**Comment by dardanbehluli** [comment]
What I did about this (thanks Gemini) is have such configuration on my akvorado.yaml:
  - asns:
     64520: vlan100-xyzMall
and
  - networks:
    10.22.80.0/20:
      asn: 64520
      name: vlan100-xyzMall
      role: clients
this way I see no more 0:??? at my Visualize tab.
Dardan from Ipko
